Patches/MeetingHudPatch.cs

```diff
@@ -1515,6 +1515,7 @@ public static void Postfix()
     {
         MeetingStates.FirstMeeting = false;
         Logger.Info("------------End Meeting------------", "Phase");
+        Quizmaster.OnMeetingEnd();
         if (AmongUsClient.Instance.AmHost)
         {
```

Comment:
Isn't it better to move `Quizmaster.OnMeetingEnd();` after `if (AmongUsClient.Instance.AmHost)`